src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/DashboardItemOrder.tsx

```diff
@@ -109,18 +92,7 @@ export const DashboardItemOrder = ({
           <div className="dashboard-item__block__status">
             <Icon name={IconTypeEnum.SCHOOL} />
             <div>
```

Comment:
Is this `<div />` really useful ? If yes, do you think we should move it into `OrderStateMessage` ?

Reply:
it was useless